src/styles/utils/_vars.scss

```diff
@@ -1 +1,3 @@
-$c-gray: #eee;
+$background-color: #fff;
```

Comment:
- all of the colors used should be put into variables
- don't name color variables with names like `main-color` but rather describe how the color looks like, for example `light-blue`